https_client.cpp

```diff
@@ -73,8 +73,17 @@ int main(int argc, char *argv[]) {
     request_stream << "Accept: */*\r\n";
     request_stream << "Connection: close\r\n\r\n";
 
+    auto size_to_send = request.size();
+    boost::asio::async_write(socket, request,
```

Comment:
I like making a namespace alias `net` for `boost::asio` and using that for all APIs that map to the Networking TS